### PR TITLE
7.9.7 crashes fixes

### PR DESCRIPTION
**Pull request by jigar-f**
Looks like Top crash is not showing in the latest version, Looks like my last fix worked, but it is Still too early to tell 

@atavism Any idea on this crash
https://play.google.com/console/u/0/developers/4642290275832863621/app/4973965144252805146/vitals/crashes/ad1e399ffe60cfa9e1ae75817567584b/details?isUserPerceived=true&days=28&versionCode=476977755

**Fixed**

- https://play.google.com/console/u/0/developers/4642290275832863621/app/4973965144252805146/vitals/crashes/84ca76c743d26818844192ff088552e9/details?isUserPerceived=true&days=28&versionCode=476977755
- https://play.google.com/console/u/0/developers/4642290275832863621/app/4973965144252805146/vitals/crashes/2482225f21cf7774bca0e8a6e1d45ce3/details?isUserPerceived=true&days=28&versionCode=476977755
- https://play.google.com/console/u/0/developers/4642290275832863621/app/4973965144252805146/vitals/crashes/fa9dbd0b646f0c34f15773d8bb193cd8/details?isUserPerceived=true&days=28&versionCode=476977755

### CODE REVIEW

**Comment by jigar-f** [comment]
@atavism I think you should merge this changes soon and upload new build.

File: android/app/src/main/kotlin/org/getlantern/lantern/MainActivity.kt [hunk 1]
```diff
@@ -274,25 +287,33 @@ class MainActivity :
...
+                if (!isFinishing) {
+                    accountInitDialog?.show()
+                }
             }
 
             AccountInitializationStatus.Status.SUCCESS -> {
-                accountInitDialog?.let { it.dismiss() }
+                if (!isFinishing) {
+                    accountInitDialog?.dismiss()
+                }
```
Comment (atavism) [inline android/app/src/main/kotlin/org/getlantern/lantern/MainActivity.kt:298]: Thanks for making these changes! Just curious, do we need this account initialization process anymore? On my related PR, I removed the step setting this to "initializing" after Lantern starts. Now for fresh installs, the dialog no longer appears, and Lantern works seamlessly, leading to an overall better experience when the app is first opened

**Comment by atavism** [comment]
> I think you should merge this changes soon and upload new build.
I think we should work more on latest build crash rate, and not merge any big changes let's wait till things get stable. we want to bring down crash rate, I fixes most of issues on new build, I need help with one, as you see above.

You’re suggesting we shouldn’t merge any big changes and focus on stabilizing the build, but you also mention that most of the issues are fixed and that we should merge the changes soon. Could you clarify if the changes are ready for review, or if you think we need more work before moving forward?

File: android/app/src/main/kotlin/org/getlantern/lantern/MainActivity.kt (hunk 1 above)
Comment (jigar-f) [reply android/app/src/main/kotlin/org/getlantern/lantern/MainActivity.kt:298]: Once we merge your changes, we do not need it.

**Comment by jigar-f** [comment]
~~I think we should work more on latest build crash rate, and not merge any big changes let's wait till things get stable. we want to bring down crash rate, I fixes most of issues on new build, I need help with one, as you see above.~~

Changes are indeed ready, Once ready we can merge for sure, But Did you get a chance to look at the crash, I mentioned at top?

**Comment by atavism** [comment]
Thanks! When it comes to the crash above, it looks like it's happening in the Go code, but there's limited details available from the stack trace.

I just realized we aren't using in Sentry *in Go* on Android. How about we start using it there? https://github.com/getlantern/lantern-client/pull/1211 

That should give us some more visibility into the crash. I've also added code to recover and log any panics that occur when starting Lantern

**Comment by jigar-f** [comment]
Thanks for looking!

**Pull request merged**